cilium/.gitignore

```diff
@@ -1 +0,0 @@
-cilium
```

Comment:
Shouldn't this be changed to `cilium-dbg` too?

Reply:
Good catch. I'll fix this up when I submit the final version with passing external workloads test :pray: